docs/README.md

```diff
@@ -17,3 +17,16 @@ make html
 
 Outputs to `build/html/index.html`
 
+## View docs web page by opening HTML in browser:
+
+First navigate to `/build/html/` folder, i.e., `cd build/html` and then run the following command:
```

Comment:
Do docs get built by default? It looks like the CMakeLists.txt defines a new target (`docs_cugraph`) which (I'm assuming?) needs to be explicitely added to a call to `make` by the user.  If that's the case, maybe the README should explain that (and possibly what new dependencies if any that need to be installed to build them)?